README.md

```diff
@@ -205,29 +127,17 @@ Example:
 
 > Mention all of the collaborators of this project.
 
```

Comment:
- [ ] Since this line is just a comment in the readme template you used that instructs you to mention all of the collaborators of this project, I suggest this comment should not show in your readme so please remove it. :pray: